config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -1959,6 +1959,20 @@
 # DEFAULT: 2 (new assignments will use 2 decimal places for grading by default)
 #assignment.grading.decimals=1
 
+# ######################################
+# SAK-41821 Student confirmation page
+# ######################################
+# Custom text shown for students on the confirmation screen, loaded by default from properties but it can be overriden here
+# assignment.customConfirmMessage=You are about to submit this submission for grading. Please check that the following items have been completed before submitting.
+
+# If set to true, submissionConfirmCheck items will have to be checked to submit
+# DEFAULT: false - submissionConfirmCheck items will appear as a bullet list
+# submissionConfirmChecks=true
```

Comment:
prefix this option with "assignment."

Reply:
Thanks! Just updated this and also solved some accessibility issues.